crates/orderbook/openapi.yml

```diff
@@ -579,11 +579,16 @@ components:
         Provides the additional data for ethflow orders
       type: object
       properties:
-        is_refunded:
+        refunded_status:
           description: |
-            Provides a flag that shows whether the refunding service has 
-            refunded the order, in case it was not settled onchain
-          type: boolean
+            Describes the status of the refund
+          type: string
+          enum: 
+            [
+              NonRefundable,
+              NotYetRefunded,
+              Refunded,
+            ]
```

Comment:
Can we add the `refundTx`?

Reply:
No, this will require even more time. It's not planned for the first version. Wasn't this feature something good to have?

Reply:
How will we detect in the frontend then when the refund has been initiated?
Will we have to watch for an onchain event on user's address?

Reply:
I think showing a pending refund is overkill. I would not show this in the first version of the UI.

Reply:
We can reiterate, sure, but we find it kind of relevant to have, and since we said that the backend will end way earlier than the frontend, wouldn't it be good to have for the FE launch day? (we reiterate after the 18th)

We are happy just with the most simple version, where the service that do automatic refunds, would persist the tx hash in the moment they post this transaction. We don't need to handle any edge case, just this. 

I might be missing it, but why do you think this is so problematic? 
If it is problematic, we can take it out, is just that I still don't think i understood why this is bringing complexities.

Reply:
So what we can do: 
If we are refunding an order, we will store the tx hash of the refund tx. 
But it might happen that in the same time - while our tx is in flight - something else refunds the user order, i.e. the order will be refunded by the user. Then our link to the tx_hash will show the wrong information.
If this is okay with everyone, then we can go for it. It just must be clear in the front-end that this is our refund attempt and not that this tx is where the refund really happened. (though 98% it's the same thing).
In order to get this information, we would do some more event parsing which is additional work.